src/uwtools/api/config.py

```diff
@@ -97,8 +96,7 @@ def realize(
         input_format=input_format,
         output_file=output_file,
         output_format=output_format,
-        values=_ensure_config_arg_type(values),
-        values_format=values_format,
+        supplemental_configs=supplemental_configs,
```

Comment:
`values_format` has been removed entirely. Users will have to provide recognizable file formats.

Reply:
I haven't reviewed the unit tests yet, but if there's not one already, it might be nice to have a test that demonstrates that, if a supplemental config is provided that has no or an unrecognizable suffix, that we fail in an appropriate way that will inform the user of the problem.